### PR TITLE
test: PartialExport_Widgets_spec - updated test validation logic

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Spec - cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts

RCA:
Comparison of this file post export is failing this file needed change.
closeModal('Insert_Modal')) is changed to closeModal(Insert_Modal.name)) and
showModal('Delete_Modal')) to showModal(Delete_Modal.name))

Solution:
To overcome this issue of frequent updation of file we have come up with a solution to validate the partial import post export in a different way.
1. We are creating a separate page and importing the partially exported file 
2. We are doing widget validation based on the selection we did during our export
3. We are checking the properties of the widgets in the process in the new page

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts [hunk 1]
```diff
@@ -20,14 +21,52 @@ describe(
       partialImportExport.OpenExportModal();
     });
 
-    it("1. Should export all the widgets", () => {
-      partialImportExport.ExportAndCompareDownloadedFile(
-        "widgets",
+    it("1. Import Downloaded file and Verify all the widgets and its properties", () => {
```
Comment (trishaanand) [inline app/client/cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts:24]:
```suggestion
    it("1. Import Downloaded file and Verify all the widgets and their properties", () => {
```

File: app/client/cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts [hunk 2]
```diff
@@ -20,14 +21,52 @@ describe(
       partialImportExport.OpenExportModal();
     });
 
-    it("1. Should export all the widgets", () => {
-      partialImportExport.ExportAndCompareDownloadedFile(
-        "widgets",
+    it("1. Import Downloaded file and Verify all the widgets and their properties", () => {
```
Comment (trishaanand) [inline app/client/cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts:24]: Isnt the test case first exporting and then importing? The title seems to suggest that we already have a json file before we start the test.

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/PartialImportExport/PartialExport_Widgets_spec.ts]: updated test name

File: app/client/cypress/support/Pages/PartialImportExport.ts [hunk 3]
```diff
@@ -175,4 +185,34 @@ export default class PartialImportExport {
...
+    });
+
+    this.agHelper.AssertElementEnabledDisabled(
+      this.locators.export.modelContents.exportButton,
+      0,
+      false,
+    );
+    this.agHelper.GetNClick(this.locators.export.modelContents.exportButton);
+    this.agHelper.FailIfErrorToast(
+      "Error exporting application. Please try again.",
```
Comment (ApekshaBhosale) [inline app/client/cypress/support/Pages/PartialImportExport.ts:215]: can we move this to messages file please? @NandanAnantharamu

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/fixtures/PartialImportExport/WidgetsExportedOnly.json
Comment (ApekshaBhosale) [inline app/client/cypress/fixtures/PartialImportExport/WidgetsExportedOnly.json:1]: why do we need this file now? @NandanAnantharamu

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/fixtures/PartialImportExport/WidgetsExportedOnly.json
Comment (NandanAnantharamu) [reply app/client/cypress/fixtures/PartialImportExport/WidgetsExportedOnly.json:1]: this file is used in another test as well , i updated with latest json file after exporting the partial app.

File: app/client/cypress/support/Pages/PartialImportExport.ts (hunk 3 above)
Comment (NandanAnantharamu) [reply app/client/cypress/support/Pages/PartialImportExport.ts]: done

**Review by ApekshaBhosale — DISMISSED** [review]

**Review by ApekshaBhosale — APPROVED** [review]

**Review by ApekshaBhosale — APPROVED** [review]

**Pull request merged**